client/lib/analytics/ad-tracking.js

```diff
@@ -780,6 +802,22 @@ function recordProduct( product, orderId ) {
 			}
 		}
 
+		// Facebook
```

Comment:
Tracking a Purchase both for the entire order and for each product may cause issues, so I advise you confirm/sync the change with Ori. A few weeks ago we removed per-product tracking, and we added it per order, in the idea that we want to consider one conversion for each total order. Before that change, the numbers looked odd/unrealistic and, after the change, they started getting fixed  (quote from Ori: "I see much less conversions with an average value that makes sense").

If you apply this change, the numbers will start looking odd again, since for one single cart we will get minimum 2 conversions. Of what I know, the order_id is NOT used to group the reporting (I have this information in an e-mail).

Also, I would consider asking about skipping all the products that cost 0 (free_domain, if product.cost < 0.01) or less (i.e. the credits part in the cart).

The summary: please talk to Ori and Abraham to find some common ground and make sure they'll still understand their own reporting after this change.

Reply:
Thank you. I checked with Ori and I'm excluding the per-product reporting for now. We can revisit in the future if per-product reporting becomes desirable. For now I am commenting that section of code out and noting the issue that you raised here.